util/noeintr.c

```diff
@@ -42,11 +42,11 @@ noeintr_write(int d, const void * buf, size_t nbyte)
 		 * pointer and adjust the remaining write length.
 		 */
 		p += lenwrit;
```

Comment:
I'd cast lenread to (size_t) both times, even if clang doesn't care about pointer arithmetic.